src/sentry/api/bases/organization_events.py

```diff
@@ -240,33 +240,39 @@ def handle_on_demand(self, request: Request) -> tuple[bool, MetricSpecType]:
 
         return use_on_demand_metrics, on_demand_metric_type
 
-    def get_split_decision(self, has_errors, has_transactions_data):
+    def save_split_decision(self, widget, has_errors, has_transactions_data, organization, user):
```

Comment:
The diff looks weird, but I deleted the `get_split_decision` helper just so we can have more control over setting the values when analyzing the decision.